File: TimeTrackerBack/TimeTracker/Models/VacationRequest.cs
namespace TimeTracker.Models
{
    public class VacationRequest
    {
        public int Id { get; set; }
        public int RequesterId { get; set; }
        public User Requester { get; set; }
        public string InfoAboutRequest { get; set; }
        public string Status { get; set; }
        public List<ApproverNode> ApproversNodes { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
    }
}

File: TimeTrackerFront/time-tracker-front/src/Redux/Types/VacationRequest.ts
import {User} from "./User";
import {ApproverNode} from "./ApproverNode";

export interface VacationRequest {
    id: number,
    requesterId: number,
    infoAboutRequest: string,
    status: string,
    startDate: Date,
    endDate: Date
    requester: User,
    approversNodes: Array<ApproverNode>
}